citations-impl/impl/src/java/org/sakaiproject/citation/impl/DbCitationService.java

```diff
@@ -569,6 +573,9 @@ else if (currentH1Node!=null){
 						currentH1Node.addChild(citationCollectionOrder);
 					}
 				}
+				else if (citationCollectionOrder.getSectiontype().equals(CitationCollectionOrder.SectionType.DESCRIPTION)){
+					lastNode.addChild(citationCollectionOrder);
```

Comment:
This has the risk again if the DB structure isn't correct of a NPE because a description might be the first element.